docs/UserGuide.md

```diff
@@ -10,19 +10,23 @@ title: User Guide
 --------------------------------------------------------------------------------------------------------------------
 ## GradPad
 
-GradPad is a one-stop solution to module management for Computer Science Undergraduates.
+We are a team of **Computer Science Undergraduates** working together to solve **module management** and **graduation planning** problems
+faced by fellow Computer Science students. Below is the detailed **User Guide** for effective use of GradPad which can help you
+ease and improve your **modules planning process**.
+
+GradPad is a one-stop solution to **module management** for Computer Science Undergraduates.
 Planning for modules has always been a tedious process but it does not have to be.
 
 The current approach to planning and tracking graduation requirements is to open up tabs after tabs of NUS resources which can
```

Comment:
I think Serene mentioned that we can list examples of the "NUS resources" we are talking about. Maybe we can say NUSMods and/or faculty websites?

Reply:
Yup I've added them in. Should I include hyper links to NUSMods and website of faculty of computing?

Reply:
Hmm, I didn't think of that but since you've suggested it, why not? Haha I think there's no harm and it might be good :D